docs: add steps for project development setup (#325)
appwrite/console: CONTRIBUTING.md

## Development

### 1. Clone the repository with git 
```bash
git clone https://github.com/appwrite/console.git appwrite-console
```

### 2. Install dependencies with npm
```bash
npm install 
```

### 3. Setup environment variables 

> add .env file using .env.example file as a template in project root directory

Reviewer: ```suggestion
Add a `.env` file by copying the `.env.example` file as a template in the project's root directory.
```
